File: api/v2010/message/create-test-sms-messages-example-1/create-test-sms-messages-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

message = @client.messages.create(
                             body: 'All in the game, yo',
<<<<<<< Updated upstream
                             from: '+15017122661',
=======
                             from: '+5571981265145',
>>>>>>> Stashed changes
                             to: '+5571981265131'
                           )

puts message.sid

File: api/v2010/message/create-test-sms-messages-example-1/create-test-sms-messages-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var message = MessageResource.Create(
            body: "All in the game, yo",
            from: new Twilio.Types.PhoneNumber("+5571981265145"),
            to: new Twilio.Types.PhoneNumber("+5571981265131")
        );

        Console.WriteLine(message.Sid);
    }
}
